docs/guides/key-manager/execute-relay-transactions.md

```diff
@@ -292,8 +320,9 @@ To execute a signed transaction, ABI payload requires:
 
 - the **KeyManager contract address**
 - the **transaction ABI payload**
-- the **signed transaction payload**
 - the **nonce** of the controller key which signed the transaction.
+- the **validity timestamps** for the execution of the relay call.
```

Comment:
I wouldn't add the link because the section where validity timestamps it is not only related to validity timestamps, it is the **How to sign relay transactions?** section.